installer/dbscript/dbscript-2.sql

```diff
@@ -4278,7 +4281,9 @@ VALUES (1, 0, 1, 0, 0, @apiv2_time_time_sheet_config_data_group_id, @admin_role_
        (1, 1, 1, 1, 1, @apiv2_time_projects_data_group_id, @admin_role_id),
        (1, 0, 0, 0, 0, @apiv2_time_project_name_validator_data_group_id, @admin_role_id),
        (1, 0, 1, 0, 1, @apiv2_time_timesheets_items_data_group_id, @admin_role_id),
-       (1, 0, 1, 0, 1, @apiv2_time_timesheets_items_data_group_id, @ess_role_id);
+       (1, 0, 1, 0, 1, @apiv2_time_timesheets_items_data_group_id, @ess_role_id),
+       (1, 0, 0, 0, 0, @apiv2_time_timesheet_action_log_data_group_id, @admin_role_id),
+       (1, 0, 0, 0, 0, @apiv2_time_timesheet_action_log_data_group_id, @ess_role_id);
 
```

Comment:
Not sure whether the role based permission is correct. @RajithaKumara please check